composer.json

```diff
@@ -40,8 +40,8 @@
     "require-dev": {
         "brianium/paratest": "^4.1",
         "composer/composer": "~1.9 || ^2.0@dev",
-        "jakub-onderka/php-parallel-lint": "~1.0",
         "nikic/php-parser": "~4.0",
+        "php-parallel-lint/php-parallel-lint": "^1.2",
```

Comment:
Could we also change this in the suggest section?

Reply:
It's done now. Thanks